### ISSUE TITLE
Enable create and update on conformance resources

### ISSUE THREAD

**Issue by nkarip** [ahdis/matchbox#70]
We have been using v2.3.0 in development to validate local dev changes to an IG but using the PUT operation to over-write individual structure definitions and then run unit tests. In versions >3 the PUT operation is disabled. Now it seems we have to reload the entire IG package, which slows down development by requiring re-running the IG publisher after any change and updating the full IG. 

Would it be possible to enable create and update on the conformance resources?

**Comment by oliveregger** [comment]
hi @nkarip, thanks for the feedback. 

we changed the handing of the conformance resources in matchbox, they are now loaded directly out of the provided package and we have an engine for each package and its dependencies. this is allows us to handle validation and transformation for multiple ig version.

However I noticed also that we lost the POST/PUT behaviour :-( We will re-add the PUT behaviour. And we will need to add a parameter to control how long the engine will be cached (currently it is 10 minutes).  For POSTs we have an issue, that we don't know yet for which IG the StructureEngine belongs too, however I will look to come up with a solution ....
